### DISCUSSION TITLE
Cmd + k shortcut is already used in Firefox

### DISCUSSION THREAD

**Discussion by gjuric** [community/community#15731] [Other Feature Feedback, Questions, & Ideas] [answered]
This shortcut is being hijacked from the browser to launch the Command palette when by default Firefox jumps to the search bar with this shortcut.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼, Thanks for the feedback, we do have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.
